app/src/main/java/com/example/concordia_campus_guide/Helper/PathFinder.java

```diff
@@ -117,6 +125,20 @@ public WalkingPoint getWalkingPointCorrespondingToPlace(final Place place) {
         return new ArrayList<>();
     }
 
+    private boolean isWalkingPointAccessible(WalkingPoint point){
+        // TODO: #147 Use string def instead of an enum
```

Comment:
Is this done?

Reply:
Will be handled in improvement task #147